README.md

```diff
@@ -18,6 +18,15 @@ exit(Code::Forbidden);
 exit(Code::Unavailable);
 ```
 
+#### Python
+
+```python
+from exit import Code
+
+Code.Forbidden.exit()  # the user isn't permitted to perform this action
+Code.Unavailable.exit()  # an API this program consumes isn't available
```

Comment:
In Go, the package is always included in the code — so I named the enumeration `Code` because it would always appear as `exit.Code` (`package`.`Type`)

In this case, I think maybe `Code` on its own is too generic... Maybe the enum should be called `ExitCode`?

What do you think?

Reply:
It depends on how you import it, which has always made naming things a bit tricky. 

```python
import exit

exit.Code.Forbidden.exit()
exit.Code.OK.exit()
```

Reply:
I think I would prefer the way it's written now just because then the fully-qualified name (`exit.Code`) makes sense, although people may choose not to import use it with the fully-qualified name. This is generally the convention I think most Python libraries follow.

Reply:
There's also import aliases, although tbh I rarely see these outside of a few extremely common abbreviations:

```python
from exit import Code as ExitCode
```

Reply:
> I think I would prefer the way it's written now just because then the fully-qualified name (exit.Code) makes sense, although people may choose not to import use it with the fully-qualified name. This is generally the convention I think most Python libraries follow.

👍  I'm convinced!

Can you document it that way in the README then too?

Reply:
👀 Good point! I was going to ask Scott about that — and about the custom `exit` function. In the Go implementation, that's something I considered (`exit.With(exit.UsageError)`), but I'm figuring that CLI authors will already be familiar with their languages' standard exit functions (`os.Exit`, `std::process::exit`, etc) and code might be a tiny bit more readable by using those.

Reply:
oh, that's definitely a footgun! 😅 

If it makes more sense to have a custom exit function in Python, go for it.

Is there a way to make the enum values implicitly cast themselves to integers?

Reply:
Yes, but in this case, no:

> The optional argument arg can be an integer giving the exit status (defaulting to zero), or another type of object. ... If another type of object is passed, `None` is equivalent to passing zero, and **_any other object is printed to [stderr](https://docs.python.org/3/library/sys.html#sys.stderr) and results in an exit code of 1_**.

https://docs.python.org/3/library/sys.html#sys.exit

Reply:
This was a new learning for me - I kinda expected `sys.exit()` would try to coerce the argument into an integer, in which case we could've made it work. Alas, it does something else entirely.

Reply:
> […] and about the custom `exit` function.

When I was writing the example, the custom function _felt_ better because otherwise there's that ugly (though quite idiomatic) `.into()`. 😉